x-pack/plugins/security_solution/public/management/components/endpoint_response_actions_list/components/hooks.tsx

```diff
@@ -5,12 +5,13 @@
  * 2.0.
  */
 
-import { useCallback, useState } from 'react';
+import { useCallback, useMemo, useState } from 'react';
 import type {
   DurationRange,
   OnRefreshChangeProps,
 } from '@elastic/eui/src/components/date_picker/types';
 import type { DateRangePickerValues } from './action_list_date_range_picker';
+import { RESPONSE_ACTION_COMMANDS } from '../../../../../common/endpoint/types';
```

Comment:
@parkiino was asking me about this recently - if we kept a list of all response actions commands anywhere. I did not even think to look inside of `types` for it. We'll need to move this to some `constants` file - maybe on under `common/endpoint/service/actions`

Reply:
@paul-tavares I feel like the action list const should be within`common/endoint/constants.ts`? What do you think?

Reply:
That's fine. I only suggested a new more specific file because IMO, the current `constants.ts` is getting to big and covers const's for many different areas of the app.

Reply:
I will leave it in the types file for now. Let's agree on where to move the response actions const.  Perhaps we should break up the `common/endpoint/constants.ts` file as well?